src/main/java/frc/robot/OI.java

```diff
@@ -174,6 +179,26 @@ private void setButtonAction(String name){
         private static final Button kBuddyBalanceActivateButton = Button.B; //Activates buddy balance
         private static final Button kDownBuddyBalanceButton = Button.POVDOWN; // Lowers buddy balance lift
         private static final Button kUpBuddyBalanceButton = Button.POVUP; // Raises buddy balance lift
+        private static final Button kYellowCargo = Button.Y;
+        private static final Button kPurpleCargo = Button.X;
+
+        private static final int kArmElbowAxis = 0;
+        private static final int kArmBaseAxis = 2;
+
+        //TODO: Tune curves to driver preference
+        private static final ControlCurve kXTranslationCurve = new ControlCurve(0.65,0.05,0.75,0.1);
+        private static final ControlCurve kYTranslationCurve = new ControlCurve(0.65,0.05,0.75,0.1);
+        private static final ControlCurve kRotationCurve = new ControlCurve(1,0,1,0.1);
+ 
+        public static Supplier<Double> getArmElbowSupplier(){
+            //This axis is inverted
+            return () -> kXTranslationCurve.calculate(-kJoystick.getRawAxis(kArmElbowAxis));
```

Comment:
This should not be using the same translationcurve as the X axis.  

A control curve makes sure that the input moves slowly at first, and then quickly move as you push harder on the joystick.  
( Instead of a linear output, which will increase the same at the beginning as at the end.  )

You have a couple of choices here.  Should talk with drive team:
1) Linear output with a deadband.
2) A new control curve for ArmElbow and ArmBase
3) one new control curve for armElbow and Armbase both.

#2 might be the best choice...